sympy/printing/fcode.py

```diff
@@ -129,6 +131,33 @@ def _lead(self):
         else:
             raise ValueError("Unknown source format: %s" % self._settings['source_format'])
 
+    def _print_Symbol(self, expr):
+        if self._settings['name_mangling'] is True:
+            from sympy import symbols, ordered, Symbol
```

Comment:
do these imports need to be here? (or would e.g. `from sympy.core.symbol import Symbol` work at the beginning of this file?)